app/models/pregnancy.rb

```diff
@@ -33,6 +33,7 @@ class Pregnancy
   # Validations
   validates :created_by,
             presence: true
+  validate :pledge_sent, :check_other_validations?, if: :updating_pledge_sent?
```

Comment:
I like this a lot -- validating only when changing it to true is a really creative and clever implementation! Nicely done 💯